server/index.js

```diff
@@ -81,10 +90,39 @@ app.get('/api/products/:product_id/styles', (req, res) => {
       res.send(response.data);
     })
     .catch((err) => {
-      res.status(404).send(err);
+      res.send(`Error:: ${err}`);
+    });
+});
+app.put('/reviews/:review_id/helpful', (req, res) => {
```

Comment:
Can we change this endpoint to `/api/reviews/:review_id/helpful` to stay consistent with the others?

Reply:
Changed it.